packages/components/releaseNotes/components.md

```diff
@@ -1,6 +1,10 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages
 
+### version 5.X
+*Released*: X November 2024
+- Incorporate Recent Storage Dashboard Widget
```

Comment:
can you add the changes mentioned in the PR to this as sub bullets?

Reply:
done